blink/README.md

```diff
@@ -0,0 +1,34 @@
+# BlinkOps Integration
+
+## Overview
+
+The [Blink][1] integration enables you to:
```

Comment:
Hey @liavt - thanks for making the changes! In addition to what you currently have, It would be helpful if you could include 1-2 sentences about the integration and the value.  We typically like to have the following:

1. 1-2 sentences about the integration so customers know what it is
2. What customers will get from this integration (what you have now covers this) 
3. Data collected (What you have now covers this)

Reply:
Hi, I added some additional information about our integration